app/assets/javascripts/autocomplete.coffee

```diff
@@ -11,7 +11,7 @@ update_textfield = (list_element) ->
   $('.js-autocomplete-resource-id').val($(list_element).data('res-id'))
 
 ready = ->
-  $('.js-autocomplete-textfield').on('focus input', ->
+  $('.js-autocomplete-textfield').on('change keyup', ->
```

Comment:
Hi @tarebyte 

Would you mind elaborating a little on using `keyup` instead of `input`? :pray: 

The reason why I used `input` is to ignore control events like `command + left arrow` (jumps to the beginning of the line).